src/main/java/org/sonar/ucfg/UCFGtoProtobuf.java

```diff
@@ -38,18 +38,20 @@ public static void toProtobufFile(UCFG ucfg, String filename) throws IOException
       .setMethodId(ucfg.methodId())
       .setLocation(toProtobuf(ucfg.location()))
       .addAllParameters(ucfg.parameters().stream().map(Expression.Variable::id).collect(Collectors.toList()))
-      .addAllUcfgEntries(ucfg.entryBlocks().stream().map(BasicBlock::label).map(Label::id).collect(Collectors.toList()))
+      .addAllEntries(ucfg.entryBlocks().stream().map(BasicBlock::label).map(Label::id).collect(Collectors.toList()))
       .addAllBasicBlocks(ucfg.basicBlocks().values().stream().map(UCFGtoProtobuf::toProtobuf).collect(Collectors.toList()))
       .build();
-    protobufUCFG.writeTo(new FileOutputStream(filename));
+    try (FileOutputStream fos = new FileOutputStream(filename)) {
```

Comment:
@benzonico for my information, what the difference?

Reply:
fos will be closed as it is called within a try with resource.

Reply:
but now I'm wondering if the stream is not closed automagically by protobuf.

Reply:
it's not, so this is more correct.